### DISCUSSION TITLE
Add an option to create border or border-radius to images

### DISCUSSION THREAD

**Discussion by taszap** [GitbookIO/community#169] [Feature Requests]
### What's your feature/idea?

I would like to be able to add borders to the images in GitBook, so that they are displayed as I wish. My white images don't look good in Light version of GitBook. 

### What's your desired outcome?

Bright images that look better in Light mode

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [X] Medium
- [ ] Low

**Comment by aikoii** [comment]
Closing this in favor of the similar one https://github.com/orgs/GitbookIO/discussions/129
